### ISSUE TITLE
'main' panicked at 'Target stage does not exist: TilemapStage'

### ISSUE THREAD

**Issue by AdamWhitehurst** [MrGVSV/bevy_tileset_map#1]
Following the README and examples provided, my program builds but then panics with:
```
2022-03-20T15:11:13.053841Z  INFO bevy_render::renderer: AdapterInfo { name: "GeForce GTX 1070", vendor: 4318, device: 7041, device_type: DiscreteGpu, backend: Vulkan }
2022-03-20T15:11:13.056139Z ERROR wgpu_hal::vulkan::instance: GENERAL [Loader Message (0x0)]
        loader_create_device_chain: Failed to find 'vkGetInstanceProcAddr' in layer C:\Program Files (x86)\Overwolf\0.191.0.20\.\ow-graphics-vulkan.dll.  Skipping layer.
2022-03-20T15:11:13.056507Z ERROR wgpu_hal::vulkan::instance:   objects: (type: INSTANCE, hndl: 0x24994f05a90, name: ?)
thread 'main' panicked at 'Target stage does not exist: TilemapStage.', C:\Users\Adam\.cargo\registry\src\github.com-1ecc6299db9ec823\bevy_ecs-0.6.0\src\schedule\mod.rs:179:32
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\runyx.exe` (exit code: 101)
```
My `main.rs`: https://github.com/AdamWhitehurst/untitled/blob/main/src/main.rs

Adding the `bevy_ecs_tilemap` stage to my `app` manually then gives a 'Target stage already exists' error. Could this be a bevy bug relating to when separate crates expect the presence of a third crate's stage?

**Comment by MrGVSV** [comment]
The issue is actually due to cargo dependencies. This crate uses the published version of `bevy_ecs_tilemap` (version 0.5), while your project uses the main branch on GitHub. This means that there are actually two `TilemapStage` types— one from `bevy_ecs_tilemap` v0.5 and one from `bevy_ecs_tilemap` main.

A quick solution is to simply use v0.5. Alternatively, you could use v0.5 and apply patch it with main (not sure if that would work either, but might be worth a shot). Lastly, you could fork this crate and change its `bevy_ecs_tilemap` dependency to use main. 

I'm not too familiar with how Rust handles situations like this, but if it affects queries and such (i.e. if `TilePos` (0.5) ≠ `TilePos` (main)) then we might have to re-export `bevy_ecs_tilemap` so the proper version is always available.

---
Related: https://stackoverflow.com/a/44437441/11621047

**Comment by AdamWhitehurst** [comment]
Late reply for anyone else stumbling across this issue. MrGVSV's solution worked! Thanks
